### PR TITLE
Try real data 23 05

### PR DESCRIPTION
**Pull request by liviahull**
## Pull request overview

This is to fix labels on the chart with highest qualification data earnings, as they were out-of-sync. I created a column with new labels and that is on the plot now instead of having an independent vector. There is an update to statement about postcode on homepage. The title for pathways chart is changed now to show the value from input selector rather than data. 

## Pull request checklist

Please check if your PR fulfils the following:
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Tests have been run locally and are passing (`run_tests_locally()`)
- [x] Code is styled according to tidyverse styling (checked locally with `tidy_code()`)

## What is the current behaviour?

Currently labels on highest qualification chart come up out of sync if there are fewer levels than 5 displayed. This is because they come from independent vector not linked to data so I created a column in the data to be displayed on the axis. Also I changed the interactive element from pathway chart to input selector so it will always be displayed. 

## What is the new behaviour?

There are no out-of-sync labels on highest qual chart and there are no errors in the title for the pathway chart.

### CODE REVIEW

File: server.R [hunk 1]
```diff
@@ -142,14 +142,22 @@ server <- function(input, output, session) {
...
       filter(Region == input$region &
-        Sector == input$sector)
+        Sector == input$sector) %>%
+      mutate(Level = case_when(
+        Level_order == "Below level 2" ~ "Below level 2 (GCSE grades 1-3)",
+        Level_order == "Level 2" ~ "Level 2 (GCSE grades 4-9)",
+        Level_order == "Level 3" ~ "Level 3 (GCE A level)",
+        Level_order == "Level 4/5" ~ "Level 4/5 (HNC or HND)",
+        Level_order == "Level 6" ~ "Level 6 (Degree)",
+        TRUE ~ "Level 7 (Master's degree)"
```
Comment (Jkhamis) [inline server.R]: Can you call this "Level 7+ (Master's degree) - we grouped up L7 and L8 as there are a small number of PHDs in the data

**Review by Jkhamis — COMMENTED** [review]
Just needs the Level 7+ change I suggested in a comment

File: R/dashboard_text.R [hunk 2]
```diff
@@ -37,7 +37,8 @@ welcome_text <- function() {
 
 industry_overview_text <- function() {
   div(
-    "This tab provides a region-specific overview of employees who work in each industry sector. It includes information on overall levels of education and average earnings, the sub-sectors of work, and subject and qualification choices. It also includes a forecast of annual employment growth in the sector between 2022 and 2027. ",
+    "This tab provides a region-specific overview of employees who work in each industry sector. It includes information on overall levels of education and average earnings, the sub-sectors of work, and subject and qualification choices.
+    It also includes a forecast of annual employment growth in the sector between 2022 and 2027. In this dashboard, region is based on the home postcode of the employee.",
```
Comment (Jkhamis) [inline R/dashboard_text.R:41]: I also emailed over changes to the median earnings text - did you include those? Can't see them here. Cheers.

Comment (liviahull) [reply R/dashboard_text.R:41]: @Jkhamis I will include them

File: server.R (hunk 1 above)
Comment (liviahull) [reply server.R]: ok

**Review by Jkhamis — APPROVED** [review]

**Pull request merged**